_rules/links-identical-name-equivalent-purpose-b20e66.md

```diff
@@ -375,15 +375,16 @@ These `span` elements do not have a [semantic role][] of link.
 These links have empty [accessible names][accessible name].
 
 ```html
-<a href="http://facebook.com"></a> <a href="http://twitter.com"></a>
+<a href="https://act-rules.github.io/"></a> <a href="https://www.w3.org/community/act-r/"></a>
 ```
 
 #### Inapplicable Example 5
 
 These image links have empty [accessible names][accessible name].
 
 ```html
-<a href="http://facebook.com"><img src="facebook.jpg"/></a> <a href="http://twitter.com"><img src="twitter.jpg"/></a>
+<a href="https://act-rules.github.io/"><img src="/test-assets/shared/act-logo.png"/></a>
+<a href="https://www.w3.org/community/act-r/"><img src="/test-assets/shared/act-logo.png"/></a>
```

Comment:
We should ditch this example. It fails the SC.

Reply:
Yeah, I noticed that. Same goes for https://act-rules.github.io/rules/fd3a94#inapplicable-example-5 which I also updated here.
I considered this was maybe unrelated with the PR, but I'm glad to take the opportunity to ditch them…